packages/browser-wallet/src/popup/pages/AddTokens/AddTokens.scss

```
@@ -0,0 +1,42 @@
+.tokens {
+    &__add {
+        &__container {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            height: 100%;
+            padding: rem(15px) rem(10px);
+        }
+
+        &__element {
+            padding: rem(10px) rem(20px);
+            margin: 0 rem(-10px);
+            display: flex;
+            border-bottom: rem(1px) solid $color-lines;
```

I think this looks nicer:

```suggestion
            padding: rem(10px) rem(10px);
            display: flex;
```

---

Yes should be the same.